code/src/components/Nav.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+export const Nav = () => {
+  return (
+    <nav className="nav">
+      <h1>
+        <NavLink to="/">
+          What's on: Sweden
+        </NavLink>
+      </h1>
+
+      <ul>
+        <li>
+          <NavLink to="/">
+            Now Playing
+          </NavLink>
+        </li>
+        <li>
+          <NavLink to="../top">
```

Comment:
There shouldn't be any `..` in path over here. Paths are absolute and it's `react-router-dom` job to understand how to use them. As you can see inside dev tools in Elements tab, those dots were deleted by React anyway

Reply:
removed.